src/lesson_7/script.js

```diff
@@ -1,72 +1,137 @@
-// Cтворити функцію, яка створює та реєструє нові команду у автопробізі. Під час виклику функції потрібно передати дані про команду - назва команди, ім'я водія, рік народження водія, марка автомобіля. спонсор команди та чи водій допущений до автопробігу
-
-
-// Запитати у користувача дані (через prompt та циклічно), про нового користувача у системі та зберегти його в об'єкт. Дані про користувача - логін, пароль, місто, країна, стать, вік
-
-const bert = prompt("Напишите свой логин, пароль, город");
-console.log('your dsta =', bert)
-
-const userData = {
-    
+function carMileage(teamName, driverName, year, carBrand, teamSponsor, isAllowed) {
+    if(typeof teamName !== 'string' || teamName.length < 3
+        || typeof driverName !== 'string' || driverName.length < 5
+        || typeof year !== 'number' ||  year < 1987
+        || typeof carBrand !== 'string' || carBrand.length < 4
+        || typeof teamSponsor !== 'string' || teamSponsor.length < 5
+        || typeof isAllowed !== 'boolean'
+    ){
+        console.log('wrong parameter');
+        return;
+    }
+    const team = {
+        teamName: teamName,
+        driverName: driverName,
+        year: year,
+        carBrand: carBrand,
+        teamSponsor: teamSponsor,
+        isAllowed: isAllowed,
+    };
+
+    return team;
 }
-console.log(userData);
-
-//userData = bert;
-
-//  мои супер пупер изменения!
-    
 
+function userData () {
+    const user = {
+        login: null,
+        password: null,
+        city: null,
+        country: null,
+        gender: null,
+        age: null,
+    };
+    let value;
+
+    for (let key in user) {
+        value = prompt('enter your' + key);
+
+        if(typeof value === 'string' && value.length > 0){
```

Comment:
`prompt` will always give you a string and you can omit `value.length > 0` and leave `value`

```suggestion
        if (value) {
```